src/main/java/com/cloudbees/jenkins/support/filter/ContentFilter.java

```diff
@@ -32,19 +33,45 @@
  * Provides a strategy to filter support bundle written contents. This is primarily useful to anonymize data written
  * to the bundle, though more complex filtering can be achieved.
  *
- * @author Matt Sicker
- * @since 2.48
+ * @since TODO
  */
 public interface ContentFilter extends ExtensionPoint {
 
+    /**
+     * @return all ContentFilter extensions
+     */
+    static ExtensionList<ContentFilter> all() {
+        return ExtensionList.lookup(ContentFilter.class);
+    }
```

Comment:
Not used outside of the interface, perhaps better to put the `lookup` method directly inside the `ofAll`
Or wait Java 9 for the private method inside interface ;)

Reply:
In Scala, I would just make the companion class implement the trait/interface as well, but Java doesn't offer that. However, I did make this `all()` method to follow the standard extension point pattern.